portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,124 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  width:1000px;
+  color:#777;
+  font-family:Verdana, Geneva, Tahoma, sans-serif;
+  background-color: lightblue;
+  margin-left:auto;
+  margin-right:auto;
 }
 
-#greeting-container {
+#pix {
+    text-align:center;
+    background-color: none;
+}
+
+h1{
+    text-align: center;
+    color:#777;
+}
+
+h2{
+    text-align: left;
+    font-weight: 500;
+    color:#777;
+}
+
+#p {
+  text-align: left;
+  color: #777;
+}
+
+#p2 {
+  text-align: center;
+  color: #777;
+}
+#greeting-container, #fact-container {
   margin-top: 20px;
 }
+
+a.one:link, a.one:visited {
+  text-decoration: none;
+  background-color:lightblue;
+  color:#777;
+
+}
+
+a.one:hover, a.one:active {
+  text-decoration: initial;
+  background-color: white;
+}
+
+a.two:link, a.two:visited {
+  text-decoration: none;
+  background-color:white;
+  color:#777;
+
+}
+
+a.two:hover, a.two:active {
+  text-decoration: initial;
+  background-color: white;
+}
+
+
+
+hr{
+    border-style: solid;
+    margin-left: 0.5cm;
+    margin-right: 0.5cm;
+    color:#777;
+}
+
+/* Create two columns/boxes that floats next to each other */
+nav {
+  float: left;
```

Comment:
I definitely recommend looking into [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) when you have some time! It's sort of the newer school way to lay out elements and may help with your projects down the line.